initiateAlerter.h

```diff
@@ -0,0 +1,12 @@
+#ifndef INITIATE_ALERTER
+#define INITIATE_ALERTER
+#include "Alert.h"
+
+struct Alerter
```

Comment:
This is an interesting structure. It wraps the pointer, but the pointer is also directly used in the rest of the program. Did you mean to use a typedef?

Reply:
I recently encountered with concept of typedef in the Ganesh's class. So, when I wrote this code then, I had not thought about the typedef implementation. But in the iteration, I will try to implement typedef to make things more readable.

Reply:
> This is an interesting structure. It wraps the pointer, but the pointer is also directly used in the rest of the program. Did you mean to use a typedef?

I did not know about the typedef concept when I wrote this code. But recently, Ganesh introduced it in the session. In the next iteration, i will see how I can use it.

Reply:
- typedef done

- de-allocation of dynamically allocated memory done.